vagrant.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+
+is_running() {
+    status=$(vagrant status | grep fedora_atomic | awk '{print $2}')
+    if [ ${status} == "running" ]; then
+        echo ""
+        echo "'${BOX}' is already running.  Perform a 'vagrant halt|destroy ${BOX}'"
+        echo ""
+        exit 1
+    fi
+}
+
```

Comment:
Maybe add some logic before running the bulk of the script to validate that the value of `${BOX}` is supported?  Some usage feedback with supported values would go nicely here, too.